default.nix

```diff
@@ -1,14 +1,6 @@
 { pkgs ? import <nixpkgs> {} }:
 
-rec {
-  home-manager = import ./home-manager {
-    inherit pkgs;
-    path = ./.;
```

Comment:
Should be sufficient to change this buggy line to `path = builtins.toString ./.;`.

Reply:
toString is a top-level function in nix:
```
nix-instantiate --eval -E 'toString 3'
```

Reply:
Sorry, yeah it is better with `path = toString ./.;`.

In any case, with this change I believe the `nix-shell $HM_PATH -A install --run 'home-manager switch'` should do the right thing.

Reply:
I added `path = toString ./.;` to master for now to fix the immediate copy bug.